Documentation/typo3cms/Developers/Index.rst

```diff
@@ -0,0 +1,196 @@
+.. _resources-developers:
+
+
+.. ------------------------------------------------------------
+.. todos:
+.. Look through the following resources if they are up-to-date
+.. and can be used or update / migrate / delete them
+..
+.. - Wiki: https://wiki.typo3.org/Extension_Developers_Guide
+.. - Wiki: https://wiki.typo3.org/Extension_Development
+.. -------------------------------------------------------------
+
+========================
+Resources for Developers
+========================
+
+This page contains some resources for TYPO3 developers including information
+about **extension development**, **site package extension development** and
+**core development**.
+
+It contains links to official TYPO3 docs. However, if the official docs are
+outdated and there are newer resources available elsewhere (e.g. on blogs),
+a link to these is supplied.
+
+You can assume that whatever starts with
+`http://docs.typo3.org/typo3cms` and is linked from this page is official
+TYPO3 documentation.
+
+.. _resources-developers-extension:
+
+Extension Development
+=====================
+
+.. _resources-developers-extension-getting-started:
+
+Getting started
+---------------
+
+For getting started with TYPO3 development, please ask for up-to-date
+resources in the Slack channel **#typo3-cms** or **#typo3-documentation**
+(register for Slack here: https://my.typo3.org/)
```

Comment:
I find the site quite confusing, shouldn't we link to https://my.typo3.org/index.php?id=35 as it's hard to find this page.

Reply:
ok